app/models/mentor_feedback_request.rb

```diff
@@ -0,0 +1,38 @@
+class MentorFeedbackRequest
```

Comment:
Can we put this file somewhere that's not in `models`? I really like that just being for (solid) DB objects, rather than helper models. But I don't know where best to put them. I like how we've split out `Policies` for example. Thoughts?

Reply:
I'm keeping it in models for now as it seems like something that we really have as a model. We create a mentor feedback request and we check whether we want to accommodate that request. It also seems like a good candidate to be saving in the database in the future.

Just to ask, what is your aversion with the models folder containing classes that don't have corresponding db tables?